jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Components.java

```diff
@@ -25,6 +25,10 @@ public interface Components
 {
     ByteBufferPool getByteBufferPool();
 
+    ByteBufferPool.Sized getInputByteBufferPool();
```

Comment:
`getByteBufferPool()` should be removed, but that could be done in another PR and the method be just deprecated for now.

Reply:
Removed and replaced all its usages with get Input/Output equivalent.